compiler/src/dmd/funcsem.d

```diff
@@ -2912,6 +2912,7 @@ bool setUnsafePreview(Scope* sc, FeatureState fs, bool gag, Loc loc, const(char)
         {
             import dmd.func : AttributeViolation;
             sc.func.safetyViolation = new AttributeViolation(loc, msg, arg0, arg1, arg2);
+            sc.func.hasSafeViolations = true;
```

Comment:
```
compiler/src/dmd/funcsem.d(2915,20): Error: no property `hasSafeViolations` for `(*sc).func` of type `dmd.func.FuncDeclaration`
compiler/src/dmd/func.d(193,14):        class `FuncDeclaration` defined here
```
assuming this is intentional, please leave it for a followup PR.

Reply:
Whoops, that wasn't supposed to be part of this commit